### PR TITLE
contribution docs

### PR DESCRIPTION
**Pull request by revanthstrakz**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Manual test
- [x] 👍 yes
- [ ] 🙅 no

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 🙅 no documentation needed

### CODE REVIEW

File: docs/contribution-guide/2-creating-pr.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+date: 2024-05-10
+category: document
+author: Revanth Rao Thopucherla
+order: 2
+
```
Comment (cpunt) [inline docs/contribution-guide/2-creating-pr.md:6]: Delete space

File: docs/contribution-guide/2-creating-pr.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+![alt text](images/branches.png)
+7. **Describe your changes**: Provide a title and description for your pull request. Click `Create pull request` to submit it.
+8. **Review and merge**: Assign the relevant person to review. Your pull request will be reviewed by the maintainers. Once approved, it will be merged into the main branch.
+9. **Delete your branch**: After your pull request is merged, you can delete your branch.
+
+## Video Walkthrough (Optional)
+
+<video src="/contributing-video.mp4"  height="240" controls></video>
+
+## End Credits
```
Comment (cpunt) [inline docs/contribution-guide/2-creating-pr.md:33]: I would call this conclusion

File: docs/contribution-guide/3-creating-issues.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+8. **Attach Files (Optional)**
+   - If you need to include screenshots, logs, or any other relevant files, you can drag and drop them or click the "Attach files" link at the bottom of the issue description area.
+   ![alt text](images/github-issue-attach1.png)
+
+9. **Submit the Issue**
+   - Once you've filled in all the necessary information, click the green "Submit new issue" button at the bottom of the page.
+   ![alt text](images/github-issue-submit.png)
+
+Congratulations! You've successfully created a new issue in your GitHub repository. This issue will now be visible to your team members, and you can track its progress, assign it to specific individuals, and collaborate on resolving it.
```
Comment (cpunt) [inline docs/contribution-guide/3-creating-issues.md:46]: I would add conclusion header here to align with the previous article

File: docs/contribution-guide/index.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+date: 2024-05-10
+category: document
+author: Revanth Rao Thopucherla
+order: 999
```
Comment (cpunt) [inline docs/contribution-guide/index.md:5]: Can we actually get the "Test Markdown" component at the bottom. So maybe set to 998 and 999 :)
Screenshot 2024-05-30 at 13 21 43

**Review by cpunt — CHANGES_REQUESTED** [review]
Nice work Revanth on cleaning up the commit history. Just a few final touches I noticed but looks really good overall!

File: docs/contribution-guide/2-creating-pr.md (hunk 1 above)
Comment (revanthstrakz) [reply docs/contribution-guide/2-creating-pr.md:6]: done

File: docs/contribution-guide/2-creating-pr.md (hunk 2 above)
Comment (revanthstrakz) [reply docs/contribution-guide/2-creating-pr.md:33]: yeah co pilot suggested conclusion but i went for end credits ;) . Done

File: docs/contribution-guide/3-creating-issues.md (hunk 3 above)
Comment (revanthstrakz) [reply docs/contribution-guide/3-creating-issues.md:46]: done

File: docs/contribution-guide/index.md (hunk 4 above)
Comment (revanthstrakz) [reply docs/contribution-guide/index.md:5]: done

**Review by cpunt — COMMENTED** [review]
Lgtm!

**Review by lePereT — APPROVED** [review]
@cpunt is happy with this, so i am happy with this

**Review by cpunt — APPROVED** [review]

**Pull request merged**